packages/nextjs/pages/index.tsx

```diff
@@ -281,15 +405,25 @@ const Home: NextPage<{ stats: Stats }> = ({ stats }) => {
 };
 
 export const getStaticProps: GetStaticProps<{ stats: Stats }> = async () => {
-  const res = await fetch(`${process.env.BG_BACKEND_API}/api/stats`);
+  //const res = await fetch(`${process.env.BG_BACKEND_API}/api/stats`);
```

Comment:
We should the env var!

Maybe you are having issues setting it up locally? Creating a `.env` file with the value + restarting the frontend (`yarn start`) should suffice

Reply:
I was not sure if Vercel test environment had env var defined, fixed now! 👍